scripts/qa_crowbarsetup.sh

```diff
@@ -2108,8 +2113,14 @@ function custom_configuration
         ;;
         rabbitmq)
             if [[ $hacloud = 1 ]] ; then
-                proposal_set_value rabbitmq default "['attributes']['rabbitmq']['ha']['storage']['mode']" "'drbd'"
-                proposal_set_value rabbitmq default "['attributes']['rabbitmq']['ha']['storage']['drbd']['size']" "$drbd_rabbitmq_size"
+                if [[ "$want_database_sql_engine" -eq "mysql" ]] ; then
```

Comment:
Hmm, how does rabbitmq depend on the database backend? It doesn't - maybe this should be dependent on something else, like the cloudversion for example